### PR TITLE
Fixed vulnerabilities for shelljs

### PR DESCRIPTION
**Pull request by SvetlanaMaliugina**
**Task name**: common-npm-packages/securefiles-common

**Description**: 
- Fixed minor vulnerabilities by running the npm audit fix command.
- azure-pipelines-task-lib version was updated too because the previous version used shelljs 0.8.4

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** [[Governance alerts](https://github.com/microsoft/build-task-team/issues/3074)] Common/securefiles-common (shelljs)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by kirill-ivlev — APPROVED** [review]

File: common-npm-packages/securefiles-common/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tasks-securefiles-common",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (DenisRumyantsev) [inline common-npm-packages/securefiles-common/package.json:3]:
```suggestion
  "version": "2.207.0",
```

File: common-npm-packages/securefiles-common/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "@types/node": "^10.17.0",
     "@types/q": "^1.5.4",
     "azure-devops-node-api": "10.2.2",
-    "azure-pipelines-task-lib": "^3.1.0"
+    "azure-pipelines-task-lib": "3.3.1"
```
Comment (DenisRumyantsev) [inline common-npm-packages/securefiles-common/package.json:23]:
```suggestion
    "azure-pipelines-task-lib": "^3.3.1"
```

**Review by DenisRumyantsev — APPROVED** [review]

**Comment by SvetlanaMaliugina** [comment]
@mmrazik could you please approve it?

**Comment by DmitriiBobreshev** [comment]
performed buddy testing for tasks:

DownloadSecureFileV1:
   -Unit Testing Result: all tests passed, file: [DownloadSecureFileTestsOut.txt](https://github.com/microsoft/azure-pipelines-tasks/files/8958807/DownloadSecureFileTestsOut.txt)
  - Pipeline Tests on Win, Mac and Linux machines, tests passed, logs: [DownloadSecureFilePipelineTest.zip](https://github.com/microsoft/azure-pipelines-tasks/files/8958808/DownloadSecureFilePipelineTest.zip)
 InstallSSHKeyV0 
   -Unit Testing Result: all tests passed, file: [InstallSSHKeyV0TestsOutput.txt](https://github.com/microsoft/azure-pipelines-tasks/files/8958803/InstallSSHKeyV0TestsOutput.txt)
   - Pipeline Tests on Win, Mac and Linux machines, tests passed, logs: [InstallSSHKeyV0PipelineTest.zip](https://github.com/microsoft/azure-pipelines-tasks/files/8958804/InstallSSHKeyV0PipelineTest.zip)

**Comment by DmitriiBobreshev** [comment]
retested tasks after revert, all tested passed, results: 
DownloadSecureFileV1:
- Unit Testing Result: all tests passed, file: [DownloadSecureFileTestsOut.txt](https://github.com/microsoft/azure-pipelines-tasks/files/8964896/DownloadSecureFileTestsOut.txt)
- Pipeline Tests on Win, Mac and Linux machines, tests passed, logs: [DownloadSecureFileTestPipeline.zip](https://github.com/microsoft/azure-pipelines-tasks/files/8964897/DownloadSecureFileTestPipeline.zip)
InstallSSHKeyV0
- Unit Testing Result: all tests passed, file: [InstallSSHKeyV0TestsOutput.txt](https://github.com/microsoft/azure-pipelines-tasks/files/8964899/InstallSSHKeyV0TestsOutput.txt)
- Pipeline Tests on Win, Mac and Linux machines, tests passed, logs: [InstallSSHKeyTestsPipeline.zip](https://github.com/microsoft/azure-pipelines-tasks/files/8964901/InstallSSHKeyTestsPipeline.zip)

**Pull request closed without merging**